api/models.py

```diff
@@ -39,17 +46,26 @@ class ClientsModel(db.Model):
 
     # Table attributes:
     id = db.Column(db.Integer, primary_key=True)
+    num_listings = db.Column(db.Integer)
```

Comment:
What is the purpose of this column? To count the listings associated to the client? If so, this is unnecessary and should be a function to get the (active) listings and count dynamically.